digdag-tests/src/test/java/acceptance/PgIT.java

```diff
@@ -0,0 +1,273 @@
+package acceptance;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.base.Throwables;
+import com.google.common.collect.ImmutableMap;
+import io.digdag.client.config.Config;
+import io.digdag.client.config.ConfigFactory;
+import io.digdag.standards.operator.jdbc.DatabaseException;
+import io.digdag.standards.operator.jdbc.NotReadOnlyException;
+import io.digdag.standards.operator.pg.PgConnection;
+import io.digdag.standards.operator.pg.PgConnectionConfig;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import utils.TestUtils;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.StringReader;
+import java.nio.file.Path;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+import java.util.UUID;
+
+import static com.google.common.base.Strings.isNullOrEmpty;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.stringContainsInOrder;
+import static utils.TestUtils.copyResource;
+
+public class PgIT
+{
+    private static final String POSTGRESQL = System.getenv("DIGDAG_TEST_POSTGRESQL");
+    private static final String RESTRICTED_USER = "not_admin";
+    private static final String SRC_TABLE = "src_tbl";
+    private static final String DEST_TABLE = "dest_tbl";
+
+    @Rule
+    public TemporaryFolder folder = new TemporaryFolder();
+    private String database;
+    private Properties props;
+
+    private Path root()
+    {
+        return folder.getRoot().toPath().toAbsolutePath();
+    }
+
+    @Before
+    public void setUp()
+    {
+        if (isNullOrEmpty(POSTGRESQL)) {
+            throw new IllegalStateException("Environment variable `DIGDAG_TEST_POSTGRESQL` isn't set");
+        }
```

Comment:
here should use `assumeThat(POSTGRESQL, not(isEmptyOrNullString()))` so that this test class itself is skipped if DIGDAG_TEST_POSTGRESQL is not set. DIGDAG_TEST_POSTGRESQL is not set when we want to test H2 database connectivity.